docs/guide/adding-custom-widget-placeholder-content.md

```diff
@@ -0,0 +1,126 @@
+# Adding placeholders to custom widgets
+
+Custom widgets can also take advantage of non-schema field placeholders by adding code to the main module, preparing the widget template to show the custom placeholder content, and adding the placeholder content into an accessible folder.
```

Comment:
"main module" is confusing. How about just "the widget module in question"

Reply:
Changed.